File: Charcutarie.Api/Charcutarie.Models/ViewModels/OrderItem.cs
using Charcutarie.Models.Enums;
using System;
using System.Collections.Generic;
using System.Text;
using System.Text.Json.Serialization;

namespace Charcutarie.Models.ViewModels
{
    public class OrderItem
    {
        public Product Product { get; set; }
        public MeasureUnit MeasureUnit { get; set; }
        public OrderItemStatus OrderItemStatus { get; set; }
        public OrderItemStatusEnum OrderItemStatusId { get; set; }
        public decimal Quantity { get; set; }
        public string AdditionalInfo { get; set; }
        public decimal Discount { get; set; }
        public decimal OriginalPrice { get; set; }
        public decimal PriceAfterDiscount { get; set; }
        public decimal ProductPrice { get; set; }
        public decimal? Cost { get; set; }
        public decimal? Profit { get; set; }
        public decimal? ProfitPercentage { get; set; }
        public int ItemNumber { get; set; }
        public int OrderItemId { get; set; }
        public Order Order { get; set; }
        public DateTimeOffset LastStatusDate { get; set; }
    }
}

File: charcutaria-business-manager/src/app/shared/models/orderItem.ts
export interface OrderItem {
    orderItemId: number;
    orderId: number;
    productId: number;
    product: string;
    quantity: number;
    orderItemStatusId: number;
    measureUnitId: number;
    measureUnit: string;
    measureUnitShort: string;
    createdOn: string;
    lastUpdated: string;
    additionalInfo: string;
    originalPrice: number;
    discount: number;
    priceAfterDiscount: number;
}